decentraland/unity-renderer
# feat: Passport names & lands integration + fixes

## What does this PR change?
- [x] Integrate Names and Lands into Extended Passport: We have created 2 new carousels showing nft names and lands owned by an user.
- [x] Add a NFT Details Info prompt for each owned wearable and emote item.
- [x] Several Passport's polishments.
- [x] Friendship status should be updated when opening the passport
- [x] Friendship status should be updated if changed while the passport is opened
- [x] Add loading spinners for the data loaded into the passport (avatar preview included).

## How to test the changes?
1. Go to: https://play.decentraland.zone/?renderer-branch=feat/passport-names-lands-integration&ENABLE_AVATAR_OUTLINER
3. Open the passport of an user.
4. Go to COLLECTIBLES section.
5. Notice 2 news sections have been added there: NAMES and LANDS.
6. Open the passport of a user. The friendship status should be displayed according to the current friendship status
7. Add/remove the user friendship while the passport is opened. The friendship status should be displayed according to the current friendship status
8. Check everything described in the "**What does this PR change?**" section of this PR is working.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
